host/sgx/loadelf.c

```diff
@@ -747,6 +747,7 @@ static oe_result_t _link_elf_image(
     /* Iterate through relocation records in the target image */
     relocs = (elf64_rela_t*)image->reloc_data;
     nrelocs = image->reloc_size / sizeof(relocs[0]);
+    result = OE_OK;
```

Comment:
OE follow the coding pattern of initializing result to OE_UNEXPECTED, performing checks and raising errors, and then finally setting result to OE_OK.

It would be desirable to preserve that style of coding in this function.

We could maintain a count of missing symbols and then print that, and raise an error if count is non zero.

Reply:
Thanks for the suggestion. I updated it.